hooks/useTeamRosterStats.ts

```diff
@@ -4,7 +4,8 @@ import { getQuerySeason } from '../utils/season';
 
 const useStandings = (
   league: string,
-  team: number
+  team: number,
+  SeasonType: string
```

Comment:
can we name this to just by `seasonType`

Reply:
Fixed